code/ui/QuickShop.scss

```diff
@@ -1,25 +1,24 @@
 QuickShop {
-    background-color: #0009;
-    border: 1px solid #000;
+    background-color: #000c;
+    border: 1px solid #1b1b1b;
     border-radius: 8px;
 
     font-family: Poppins;
-    max-width: 800px;
-    min-width: 600px;
+    width: 600px;
     left: 50%;
     top: 50%;
     transform: translateX(-50%) translateY(-50%);
     margin: auto;
     position: absolute;
     flex-direction: column;
 
-    color: white;
+    color: #fafafa;
     font-family: Poppins;
     font-weight: 500;
     pointer-events: all;
 
     &.hide {
-        display: none;
+        opacity: 0;
```

Comment:
I think `display` makes more sense because we don't want animations there. If you are a traitor and wanna shop, then the shop shall open immediately.

Reply:
We don't necessary need animations here I agree, although I think its better to generally use opacity and have this in any UI element over `display`.

Reply:
Hmm.. What are the reasons for that decision? Doesn't the element still renders with `opacity` while keeping undisplayed with `display` and saving performance? Or does it not matter? Just thought about it at all ^^

Reply:
Idk imo it's more useful to generally use `opacity` over `display` if we want to be able to add short animations later if we decide to so... I also think that a couple of hundred milliseconds wouldn't be a huge blocker a T trying to buy something real quick. I also don't know how big the differences in performance are and if the would make any impact tho... 
But I will revert this back to display...

Reply:
```
display:none will hide the whole element and remove that from layout space whereas visibility:hidden hides an element but take up the same space as before. Opacity can be used if you want to create transparency or fade effect.
```